app/models/involved_civilian.rb

```diff
@@ -48,6 +50,10 @@ class InvolvedCivilian < InvolvedPerson
     validates :firearm_type, presence: true, subset: { in: FIREARM_TYPES }, if: :firearm?
   end
 
+  def k12_setting?
```

Comment:
I would call this method `k12?` or `k12_campus_incident?` or `on_k12_campus?`. I don't really like the word `setting` here because it doesn't mean much.